jormungandr-lib/src/time.rs

```diff
@@ -250,6 +250,17 @@ impl From<SystemTime> for time::SystemTime {
     }
 }
 
+impl From<time::SystemTime> for SecondsSinceUnixEpoch {
+    fn from(system_time: time::SystemTime) -> Self {
+        system_time
+            .duration_since(time::UNIX_EPOCH)
+            // duration since UNIX EPOCH will never go beyond boundaries
```

Comment:
I would not say never, but 2^64 - 1 seconds is a long time ;)

Reply:
I think that this assumption was made with pre-1970 dates in mind, but yeah, it's a rather safe one and truly unrecoverable